modules/reference/pages/rpk/rpk-cluster/rpk-cluster-license-info.adoc

```diff
@@ -15,6 +17,14 @@ Expires:         Expiration date of the license.
 rpk cluster license info [flags]
 ----
 
+== Aliases
```

Comment:
Not actually sure what to make of this section, are these supposed to be an alias to the command? E.g. Use `status` instead of `info`?

Reply:
Yes. We have other instances in docs that we do this 
https://deploy-preview-835--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-txn-list/

the command support both writings